integrationtest/java/modules/orm-coordination-outbox-polling-elasticsearch/pom.xml

```diff
@@ -44,6 +44,12 @@
             <artifactId>hibernate-search-mapper-orm-coordination-outbox-polling</artifactId>
         </dependency>
 
+        <!-- Only needed here so we can require it in module info for Avro -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+        </dependency>
```

Comment:
Shouldn't this be just log4j-api?

Also... would this work if the dependency is marked as `<scope>provided</scope>`? I don't want to have to align the dependency on Avro...

Reply:
I've tried it locally, and looks like it is ok to build/run with provided scope.